ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -832,6 +976,8 @@ export const RuntimePanel = fp.flow(
   // the source of truth for these values are selectedMachine, and selectedDiskSize, as
   // these UI components are used for both Dataproc and standard VMs.
   const [selectedDataprocConfig, setSelectedDataprocConfig] = useState<DataprocConfig | null>(dataprocConfig);
+  const [selectedPdSize, setSelectedPdSize] = useState(pdSize);
```

Comment:
High-level comment on terminology:

- The old approach used persistent disks. The new approach also uses persistent disks.
- The main differences with the new approach:
  - new disks are optionally reattachable
  - old disks are set to autodelete with the VM
  - because a new disk can outlive a runtime, Leo provides some APIs for interacting with them directly

The older disks are still "persistent disks" from the GCP perspective. Pricing is the same.

There was probably some confusion when introducing the project, as this feature is typically referred to as "persistent disks". More accurately it might be called "reattachable disks" to avoid confusion with the GCE term